docs/source/devguide.md

```diff
@@ -259,54 +259,116 @@ Make sure your change
-  that it is still PEP-440 compatible.
+* In `xcube/version.py` increase version number and append a `.dev0` suffix 
+  to the version name so that it is still PEP-440 compatible.
+* Adjust version in `Dockerfile` accordingly.
 * In `CHANGES.md` add a new version headline and attach ` (in development)` to it.
 * Push changes to either master or a new maintenance branch (see above).
 * Activate new doc version on ReadTheDocs. 
 
-Go through the same procedure for all xcube plugin packages dependent on this version of xcube.
+Go through the same procedure for all xcube plugin packages 
+dependent on this version of xcube.
+
+### Release on Conda-Forge
+
+These instructions are based on the documentation at 
+[conda-forge](https://conda-forge.org/docs/maintainer/updating_pkgs.html).
+
+Conda-forge packages are produced from a github feedstock repository belonging 
+to the conda-forge organization. A repository's feedstock is usually located at 
+`https://github.com/conda-forge/<repo-name>-feedstock`, e.g., 
+https://github.com/conda-forge/xcube-feedstock.
+The package is updated by 
+* forking the repository
+* creating a new branch for the changes
+* creating a pull request to merge this branch into conda-forge's feedstock repository. 
+
+The first of these steps is usually already done. 
+You may find forks at `https://github.com/dcs4cop/<repo-name>-feedstock` . 
+
+In detail, the steps are:
+
+1. Update the dcs4cop fork of the feedstock repository, if it's not already 
+   up to date with conda-forge's upstream repository.
+
+1. Clone the repository locally and create a new branch. The name of the branch 
+   is not strictly prescribed, but it's sensible to choose an informative name like 
+   `update_0_5_3`.
+
+1. Rerender the feedstock using conda-smithy. This updates common conda-forge 
+   feedstock files. It's probably easiest to install conda-smithy in a 
+   fresh environment for this:
+
+   `conda install -c conda-forge conda-smithy`
+    
+   `conda smithy rerender -c auto`
+
+   It's also possible to have the rendering done by a bot as part of the pull request,
+   but this doesn't seem to work very reliably in practice.
+
+1. Update `recipe/meta.yaml` for the new version. 
+   Mainly this will involve the following steps:
+
+   1. Update the value of the version variable (or, if the version number 
+   has not changed, increment the build number).
+   
+   1. If the version number has changed, ensure that the build number is set to 0.
+
+   1. Update the sha256 hash of the source archive prepared by GitHub.
+
+   1. If the dependencies have changed, update the list of dependencies 
+   in the `-run` subsection to match those in the environment.yml file.
+
+1. Commit the changes, push them to GitHub, and create a pull request at the 
+   feedstock repository on conda-forge .
+
+1. Once conda-forge's automated checks have passed, merge the pull request.
+
```

Comment:
I suggest an additional step here: merge the newly-merged changes from the master branch on conda-forge back to the master branch of the dcs4cop fork. This has to be done anyway at some point before the branch for the following release is created, but doing it immediately makes the current configuration more visible on the dcs4cop repo, and avoids messy parallel branches if someone subsequently checks changes directly into dcs4cop's master.

Reply:
We could do so ... I don't think it's absolutely necessary and it would not replace the step that the author has to check that the branches on conda-forge and dcs4cop are identical, but your arguments are valid.